src/Mod/BIM/bimcommands/BimMoveView.py

```diff
@@ -35,7 +35,7 @@ class BIM_MoveView:
     def GetResources(self):
         return {
             "Pixmap": "BIM_MoveView",
-            "MenuText": QT_TRANSLATE_NOOP("BIM_MoveView", "Move view..."),
+            "MenuText": QT_TRANSLATE_NOOP("BIM_MoveView", "Move View…"),
```

Comment:
```suggestion
            "MenuText": QT_TRANSLATE_NOOP("BIM_MoveView", "Move View"),
```

or Move View to Page ?